base/applications/osk/main.c

```diff
@@ -57,6 +58,51 @@ int OSK_SetImage(int IdDlgItem, int IdResource)
     return TRUE;
 }
 
+/***********************************************************************
+ *
+ *          OSK_WarningProc
+ *
+ *  Function handler for the warning dialog box on startup
+ */
+INT_PTR CALLBACK OSK_WarningProc(HWND hDlg, UINT Msg, WPARAM wParam, LPARAM lParam)
+{
+    /* Declare and initialize the icon variable */
+    HICON hIcon;
+    hIcon = LoadIcon(Globals.hInstance, MAKEINTRESOURCE(IDI_OSK));
```

Comment:
Loading the icon should be once.

Reply:
Just remove this, you do not even use `hIcon`.

Reply:
@learn-more In Windows XP the icon is displayed at the left of the dialog text. Why do we need to remove it?
https://imgur.com/a/NOO1zBB

Reply:
Send WM_SETICON message to set the icon.

Reply:
If it was difficult for you, then please go without icon.

Reply:
Just add the icon in the dialog resource?
Don't need to manually load it.